posthog/api/test/test_preflight.py

```diff
@@ -53,6 +54,7 @@ def test_preflight_request(self):
                     "db": True,
                     "initiated": True,
                     "cloud": False,
+                    "realm": "hosted",
```

Comment:
I think it's worth adding is a test case for `hosted-clickhouse`, when do we send this attribute?